Description: Simple wrapper since classpath and LD library are already properly
 assigned at build time. Must be exucted in /usr/share/arduino
Author: Scott Howard <showard314@gmail.com>
Index: arduino-0018+dfsg/arduino
===================================================================
--- arduino-0018+dfsg.orig/arduino	2010-10-30 11:23:40.667327687 -0400
+++ arduino-0018+dfsg/arduino	2010-10-30 11:25:36.590115827 -0400
@@ -1,22 +1,8 @@
 #!/bin/sh
  
-APPDIR="$(dirname -- "${0}")"
+cd /usr/share/arduino
 
-cd $APPDIR
- 
-for LIB in \
-    java/lib/rt.jar \
-    java/lib/tools.jar \
-    lib/*.jar \
-    ;
-do
-    CLASSPATH="${CLASSPATH}:${APPDIR}/${LIB}"
-done
-export CLASSPATH
-
-LD_LIBRARY_PATH=`pwd`/lib:${LD_LIBRARY_PATH}
+LD_LIBRARY_PATH="/usr/lib/jni:${LD_LIBRARY_PATH}"
 export LD_LIBRARY_PATH
 
-export PATH="${APPDIR}/java/bin:${PATH}"
-
-java -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel processing.app.Base
+java -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel -jar /usr/share/arduino/arduino.jar
